intra/relay/templates/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+{{ range $i, $stream := .Values.streams }}
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: relay-{{ $stream }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: relay-{{ $stream }}
+  template:
+    metadata:
+      labels:
+        app: relay-{{ $stream }}
+    spec:
+      containers:
+      - name: relay
+        image: paulbouwer/hello-kubernetes:1.5
```

Comment:
@umeat Is there a docker image available for the relay code?

Reply:
Probably worth setting up a proper build script / pipeline for cmd/ntrip-mqtt-relay Docker build

Reply:
geoscienceaustralia/ntrip-mqtt-relay is on docker hub.